integrationTests/testInitializer.go

```diff
@@ -1358,6 +1385,38 @@ func CreateAndSendTransaction(
 	node.OwnAccount.Nonce++
 }
 
+// CreateAndSendTransactionFromWallet will generate a transaction with provided parameters, sign it with the provided
+// wallet's tx sign private key and send it on the transaction topic
+func CreateAndSendTransactionFromWallet(
```

Comment:
I think there is a function which does the same already. PlayerSendsTransactoin ?

Reply:
PlayerSendsTransactoin is sent from the node's address. Wanted to used that but couldn't